dapper/mods/Lorenz96s/__init__.py

```diff
@@ -1,6 +1,6 @@
 """A perfect-random version of Lorenz-96.
 
-Used by `bib.grudzien2020numerical` to study the impact of
+Used by `bib.grudzien2020numerical` to study the impact of numerical precision on
```

Comment:
I really don't see what this study has to do with "numerical precision". That refers to stuff like round-off error.